src/components/Loading.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Loading() {
+  return (
+    <p>Loading...</p>
+  )
+}
```

Comment:
Not sure that this is worth it's own component since it's just one p tag that is returned